### PR TITLE
Add trivy-operator option to pack

### PR DESCRIPTION
**Pull request by stone-z**
Adds optional support for installing `trivy-operator`, which is intended to replace `starboard`

Also bumps Kyverno to the latest version with some extra stability fixes

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/security-pack/values.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ apps:
     appName: starboard
     chartName: starboard-app
     catalog: giantswarm
-    enabled: true
```
Comment (stone-z) [inline helm/security-pack/values.yaml:59]: Self-reminder to revert this after testing

File: helm/security-pack/values.yaml [hunk 2]
```diff
@@ -75,3 +75,11 @@ apps:
     enabled: true
     namespace: security-pack
     version: 0.4.0
+
+  trivyOperator:
+    appName: trivy-operator
+    chartName: trivy-operator
+    catalog: giantswarm
+    enabled: true
```
Comment (stone-z) [inline helm/security-pack/values.yaml:83]: Self-reminder to revert this after testing

**Comment by stone-z** [comment]
Tested on a WC. Need to disable CIS benchmark exports in the exporter by default before making trivy-operator the detault. Otherwise looks ok. Will let customers test with this for feedback before defaulting to trivy-operator

**Review by fhielpos — APPROVED** [review]
Boom baby 💥

**Pull request merged**